chef-automate-ha/nested/availabilitySetsResource.json

```diff
@@ -6,7 +6,9 @@
             "type": "object"
         }
     },
-    "variables": {},
+    "variables": {
+        "provider": "[toUpper('33194f91-eb5f-4110-827a-e95f640a9e46')]"
```

Comment:
Simply it shouldn't be able to be parameterized and there shouldn't be any customization of it.